src/ui/pages/Identifiers/components/IdentifiersList.tsx

```diff
@@ -52,9 +49,7 @@ const IdentifiersList = ({
                   className="identifier-info-bottom-line"
                   data-testid={`identifier-info-${index}`}
                 >
-                  {identifier.method === IdentifierType.KERI
-                    ? i18n.t("identifiers.tab.type.keri")
-                    : i18n.t("identifiers.tab.type.didkey")}
+                  {i18n.t("identifiers.tab.type.keri")}
```

Comment:
Just a reminder for ourselves, the "type" here will probably become redundant and we need to align on removing it from the design.

![08042024131829](https://github.com/cardano-foundation/cf-identity-wallet/assets/119612231/199a57a8-0407-4e5e-a3ba-4d2f8a250996)

In the meantime, we can already remove `"didkey": "W3C did:key",` from line 653 of `src/locales/en/en.json` as it won't be used anymore.